### ISSUE TITLE
Which ARM architecture does this use?

### ISSUE THREAD

**Issue by turtleisaac** [Cryptogenic/REBot#19]
I’ve been using this for a while and something I often notice is that whichever arm architecture REBot uses isn’t the same one I’m using. I’m using ARMv5t and the only differing instruction I’ve noticed so far is that NOP is C0 46 for the architecture I use instead of the 00 BF that REBot returns. it’s an issue that catches users who don’t know to use the different NOP for this architecture off guard often on the server we use it on.

tldr: which specific ARM variant does this assemble and disassemble?
